setup.py

```diff
@@ -71,9 +73,11 @@ def run(self):
         'git://github.com/NeuralEnsemble/libNeuroML.git#egg=libNeuroML',
         'git://github.com/zopefoundation/ZODB.git#egg=ZODB',
         'git://github.com/RDFLib/FuXi#egg=FuXi-1.4.dev-r0',
+        'git+https://github.com/RDFLib/rdflib-zodb#egg=rdflib-zodb-1.1-dev',
+        'git+https://github.com/mwatts15/YAROM@dev#egg=yarom-0.7.1',
     ],
     version = '0.5.3',
-    packages = ['PyOpenWorm'],
+    packages = ['PyOpenWorm', 'PyOpenWorm.v0', 'PyOpenWorm.graphObjectAdapter'],
```

Comment:
How does the addition of packages make deployment more difficult? It is less that v0 is parallel and more that I used the original to make the new. Adapting the existing DataObject and SimpleProperty makes more sense to me than breaking everything in the process of rewrite. I could merge everything together, but that would take me longer and I wanted to bring this PR for review sooner rather than later.